src/components/SetupComponents/SetupFormBlock/index.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+
+export interface SetupFormBlockProps {
+    title: string;
+    subtitle: string;
+    children?: React.ReactNode;
+}
+
+export class SetupFormBlock extends React.Component<SetupFormBlockProps> {
```

Comment:
same here, better to use functional component according to our plan about moving Baseapp to func